### DISCUSSION TITLE
blog/create-an-rss-feed-with-nuxt-3-and-nuxt-content-v2

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#20] [Announcements]
# blog/create-an-rss-feed-with-nuxt-3-and-nuxt-content-v2

My portfolio website is built with Nuxt 3 and Nuxt Content v2. An RSS feed with my latest five blog posts is available here. In this article, you'll learn how to add an RSS feed to your Nuxt website.

https://mokkapps.de/blog/create-an-rss-feed-with-nuxt-3-and-nuxt-content-v2

**Comment by johnhom1024** [comment]
Help me a lot, I also use Nuxt Content and want to create RSS Feed,  but I don't know Nuxt Content had provided some methods to use, util I saw this post, Thank you.
